mrkwords.sh

```diff
@@ -8,8 +8,7 @@ key="$3"
 if [ -z "$key" ]; then
   word=$(shuf -n 1 < "$file" | cut -d' ' -f1)
 else
-  word=$(grep -Fw "$key" < "$file" |
```

Comment:
I'm pretty sure i remember adding that initial grep for optimization purposes, since it's a fast plaintext grep that does some initial filtering. iirc awk was slower on large datasets, so I want to keep this

Reply:
Oh yes, I can imagine grep being much more optimized than awk for plain string searching.